scripts/e2e-test.sh

```diff
@@ -47,3 +47,24 @@ operator-sdk test local ./test/e2e --namespace "$TEST_NAMESPACE" --up-local --no
 helm template deploy/helm/eunomia-operator/ \
   --set eunomia.operator.deployment.enabled= \
   --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
+
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.image.tag=dev \
+  --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl apply -f -
+
+sleep 30
+
+podname=$(kubectl get pods  -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' -n $TEST_NAMESPACE)
+
+if kubectl exec "${podname}" date -n $TEST_NAMESPACE
+   then
+     echo "Pod is Healthy"
+       else
+     echo "Pod is not Healthy"
+       exit 1
+fi
+
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.deployment.enabled= \
+  --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
```

Comment:
I think we don't need this command, because it is already done in l.47-49; instead, we want the lines that Sean suggested, to delete resources that were created in l.51-53; or am I missing something?

Reply:
`helm` command in LineNo: 51-53  will recreate all  required NS,SA,CRD's etc.... So just to make sure everything is cleaned properly using delete here.

Reply:
I completely agree we need to delete the CRDs; but for doing this, how is l.67-69 better than what Sean proposed, i.e.:

```
helm template deploy/helm/eunomia-operator/ \
  --set eunomia.operator.image.tag=dev \
  --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
```

?

Reply:
Just tested and realised proposed `helm` command also deleting all resources that are created .

Reply:
Done with suggested changes. Kindly review 
Thanks @akavel